Marlin/src/lcd/dogm/dogm_Statusscreen.h

```diff
@@ -1341,7 +1341,7 @@
     #undef STATUS_LOGO_WIDTH
   #endif
 
-  #if !defined(STATUS_HEATERS_X) && ((HAS_MULTI_HOTEND && STATUS_LOGO_WIDTH && BED_OR_CHAMBER_OR_FAN) || (HOTENDS >= 3 && !BED_OR_CHAMBER_OR_FAN))
+  #if !defined(STATUS_HEATERS_X) && ((HAS_HOTEND && STATUS_LOGO_WIDTH && BED_OR_CHAMBER_OR_FAN) || (HOTENDS >= 3 && !BED_OR_CHAMBER_OR_FAN))
```

Comment:
What does this change do? It's not obvious to me how it relates to the centering change.

Reply:
I guess this is just setting the width constants. Any idea why it previously only applied to multiple hotends? Has your testing verified it is correct with only 1?

Reply:
I don't have setup with multiple heaters, but based on the code, regardless of amount it can calculate width needed for hotends+bed+chamber+fan. That's why I enabled it to use with any amount of heaters.

Reply:
As I recall, this define is set up so that 2 or more nozzle images will be aligned right, along with bed, chamber, and fan, but a single nozzle graphic will align left after the logo image, which is what you usually want.